monai/data/transforms/transform.py

```diff
@@ -9,6 +9,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
```

Comment:
Why did you add additional 1 line? All other code files use 1 line blank between copyright and code.

Reply:
sure, I've updated the code. we might want to have some flake configurations to enforce this style.